examples/Misc/GetAllVideosAndImages.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright 2019 Google LLC
```

Comment:
2020

Reply:
+1 Across all the files that are newly added in this commit, please.

Reply:
Done.